src/config/webpack/index.js

```diff
@@ -17,6 +18,12 @@ function webpackConfig (env) {
     const entry = user.entry
     const environment = utils.getEnv(env).stringified
     environment.TEST_DIR = JSON.stringify(path.join(process.cwd(), 'test'))
+    const browserJs = path.join(process.cwd(), 'test', 'browser.js')
```

Comment:
this should use `TEST_DIR` instead (and should eventually be made an overridable option probably)